src/redux/sprints/sprints-operations.ts

```diff
@@ -27,52 +34,46 @@ export const getProjectsSprints = createAsyncThunk(
     try {
       const { data } = await axios.get(`/sprint/${projectId}`);
       return data;
-    } catch (error) {
-      dispatch(
+    } catch (error:any) {
         getError({
           error,
           cb: () => getProjectsSprints(),
           operationType: "sprint/getSprints",
         })
-      );
       return rejectWithValue(error.message);
     }
   }
 );
 
 export const changeSprintsTitle = createAsyncThunk(
   "sprint/changeTitle",
-  async (sprintId, { dispatch, rejectWithValue }) => {
+  async (sprintId:string, { dispatch, rejectWithValue }) => {
     try {
       const { data } = await axios.patch(sprintId);
       return data;
-    } catch (error) {
-      dispatch(
+    } catch (error:any) {
         getError({
           error,
-          cb: () => changeSprintsTitle(),
+          cb: () => changeSprintsTitle(sprintId),
           operationType: "sprint/changeTitle",
         })
-      );
       return rejectWithValue(error.message);
     }
   }
 );
```

Comment:
I did some research and I can propose you next solution:

At first add this types to your project. If something isn't clear for you, we'll discuss about this on meeting. 

```suggestion
import axios, { AxiosError } from 'axios';

interface IErrorBase<T> {
   error: Error | AxiosError<T>;
   type: 'axios-error' | 'stock-error';
}

interface IAxiosError<T> extends IErrorBase<T> {
   error: AxiosError<T>;
   type: 'axios-error';
}
interface IStockError<T> extends IErrorBase<T> {
   error: Error;
   type: 'stock-error';
}
```

Next you should add error handler like this:
```suggestion
export function axiosErrorHandler<T>(
   callback: (err: IAxiosError<T> | IStockError<T>) => void
) {
   return (error: Error | AxiosError<T>) => {
      if (axios.isAxiosError(error)) {
         callback({
            error: error,
            type: 'axios-error'
         });
      } else {
         callback({
            error: error,
            type: 'stock-error'
         });
      }
   };
}
```

And, finally, you should use this handler in all axios catch cases. I.e.:
```suggestion
.catch((error: Error | AxiosError) {
  if (axios.isAxiosError(error))  {
    // Access to config, request, and response
  } else {
    // Just a stock error
  }
  
  getError({
          error,
          cb: () => changeSprintsTitle(),
          cb: () => changeSprintsTitle(sprintId),
          operationType: "sprint/changeTitle",
        })
      );
      return rejectWithValue(error.message);
})
```

Reply:
Reference. Firstly you can try to find answers here: https://github.com/axios/axios/issues/3612